<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmSwitchButton</class>
 <widget class="QWidget" name="frmSwitchButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="SwitchButton" name="switchButton11">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="SwitchButton" name="switchButton12">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="SwitchButton" name="switchButton13">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="SwitchButton" name="switchButton14">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="SwitchButton" name="switchButton23">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>120</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="SwitchButton" name="switchButton22">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>70</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="SwitchButton" name="switchButton21">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="SwitchButton" name="switchButton24">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="SwitchButton" name="switchButton31">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>20</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="SwitchButton" name="switchButton33">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="SwitchButton" name="switchButton34">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>170</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="SwitchButton" name="switchButton32">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SwitchButton</class>
   <extends>QWidget</extends>
   <header>switchbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
